cli/src/main/kotlin/com/bazel_diff/cli/GetImpactedTargetsCommand.kt

```diff
@@ -38,6 +38,13 @@ class GetImpactedTargetsCommand : Callable<Int> {
     )
     lateinit var finalHashesJSONPath: File
 
+    @CommandLine.Option(
```

Comment:
I believe this should be an array? You could desire to filter out for multiple types of target. Maybe this could also be backed by an enumeration to display the allowed values

Reply:
Yeah, I have changed it to a `Set` to support multiple values. However, I was hesitant to change it to enum because in case of new types are added in the future, I expect the current code still work without need to add new values to the enum.